src/main/java/com/google/devtools/build/lib/actions/PathStripper.java

```diff
@@ -85,10 +85,12 @@ public interface ActionStager {
      *
```

Comment:
Nit:: did you want to update this comment, and comments like at like at line 112 in the new code that refer to "strip"?  Or is that premature, and/or handled by https://github.com/bazelbuild/bazel/pull/17808?

Reply:
I would keep them as is for now as they seem to be referring to the particular instances of these interfaces created in this file, which really do strip config prefixes instead of applying more general mappings.